DEVELOPMENT.rst

```diff
@@ -18,28 +16,44 @@ TODO: create readthedocs site and point to it.
 Quickstart
 ----------
 
+Intergov is tighly coupled with the countries, so if you want to start it locally
```

Comment:
At some point we will need to do a search and replace of "countries" with "jurisdictions", including environment variable names...